corehq/apps/users/management/commands/restore_domain_membership.py

```diff
@@ -0,0 +1,87 @@
+from couchdbkit import ResourceNotFound
+from django.core.management.base import BaseCommand, CommandError
+
+from corehq.apps.change_feed.consumer.feed import KafkaChangeFeed
+from corehq.apps.change_feed.topics import COMMCARE_USER, get_multi_topic_offset, \
+    get_multi_topic_first_available_offsets
+from corehq.apps.users.models import DomainMembership, CommCareUser
+
+defaults = DomainMembership().to_json()
+del defaults['domain']
+extra_keys = ['first_name', 'last_name', 'user_data']
+
+
+class Command(BaseCommand):
+    args = "since_kafka_offset"
+    help = "See http://manage.dimagi.com/default.asp?245969 for context"
+
+    def add_arguments(self, parser):
+        parser.add_argument('--print-kafka-offsets', action='store_true',
+                            help='Just print the kafka offsets')
+
+    def handle(self, *args, **options):
+        if options['print_kafka_offsets']:
+            end = get_multi_topic_offset([COMMCARE_USER])[COMMCARE_USER]
+            start = get_multi_topic_first_available_offsets([COMMCARE_USER])[COMMCARE_USER]
+            print "\nKakfa topic offset range: {} - {}".format(start, end)
+            return
+
+        if len(args) == 0:
+            raise CommandError("Usage: python manage.py resync_location_user_data %s" % self.args)
+
+        since = args[0]
```

Comment:
not sure how to determine the `since` value since we don't store the date in the change (should fix that)